c/crass/crass_rhel_8.4.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package               : crass
+# Version               : v1.0.4
+# Source repo           : https://github.com/rgrove/crass
+# Tested on             : UBI 8.5
+# Language              : Ruby
+# Travis-Check          : True
+# Script License        : Apache License, Version 2 or later
+# Maintainer            : Vathsala . <vaths367@in.ibm.com>
+#
+# Disclaimer            : This script has been tested in root mode on given
+# ==========              platform using the mentioned version of the package.
+#                         It may not work as expected with newer versions of the
+#                         package and/or distribution. In such case, please
+#                         contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
+export PACKAGE_NAME=crass
+export PACKAGE_URL=https://github.com/rgrove/crass
+
+if [ -z "$1" ]; then
+  export PACKAGE_VERSION=${1:-v1.0.4}
+else
+  export PACKAGE_VERSION=$1
+fi
+if [ -d "crass" ] ; then
+  rm -rf crass
+fi
+
+
+yum install -y git ruby procps yum-utils wget
+
+yum-config-manager --add-repo http://mirror.centos.org/centos/8/AppStream/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/PowerTools/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/BaseOS/ppc64le/os/ && yum-config-manager --add-repo http://mirror.centos.org/centos/8/virt/ppc64le/ovirt-44/
+
+wget https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official && mv RPM-GPG-KEY-CentOS-Official /etc/pki/rpm-gpg/. && rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-Official
```

Comment:
see if you can do w/o installing/enabling centos repos

Reply:
Yes..i have updated the script w/o installing/enabling centos repos..and it's working fine